src/content/blog/bitsctf2025-dfir/index.mdx

```diff
@@ -0,0 +1,378 @@
+---
+title: "BITSCTF 2025: All DFIR challenges (3/3)"
+date: 2025-02-14T08:57:16.633Z
+draft: false
+author: madarame
+tags: [dfir]
+---
+
+import css from "css/styles.css?url";
+import image from "./image.png?url";
+import flag_decrypted from "./flag_decrypted.png?url";
+
+<link rel="stylesheet" href={css}></link>
+<link
+  rel="stylesheet"
+  href="https://cdn.jsdelivr.net/npm/katex@0.16.21/dist/katex.min.css"
+  integrity="sha384-zh0CIslj+VczCZtlzBcjt5ppRcsAmDnRem7ESsYwWwg3m/OaJ2l4x7YBZl9Kxxib"
+  crossOrigin="anonymous"
+></link>
+<script
+  defer
+  src="https://cdn.jsdelivr.net/npm/katex@0.16.21/dist/katex.min.js"
+  integrity="sha384-Rma6DA2IPUwhNxmrB/7S3Tno0YY7sFu9WSYMCuulLhIqYSGZ2gKCJWIqhBWqMQfh"
+  crossOrigin="anonymous"
+></script>
+<script
+  defer
+  src="https://cdn.jsdelivr.net/npm/katex@0.16.21/dist/contrib/auto-render.min.js"
+  integrity="sha384-hCXGrW6PitJEwbkoStFjeJxv+fSOOQKOPbJxSfM6G5sWZjAyWhXiTIIAmQqnlLlh"
+  crossOrigin="anonymous"
+  onLoad="renderMathInElement(document.body);"
+></script>
```

Comment:
Don't import katex like this, we have katex by default already so u can just use $...$

I can fix it